src/mixpanel_lwt.mli

```diff
@@ -6,3 +6,31 @@ val track :
   ?options_send_immediatly:bool ->
   unit ->
   Ojs.t Lwt.t
+
+val init :
+  token:string ->
+  ?api_host:string ->
+  ?app_host:string ->
+  ?autotrack:bool ->
+  ?cdn:string ->
+  ?cross_subdomain_cookie:bool ->
+  ?persistence:string ->
+  ?persistence_name:string ->
+  ?cookie_name:string ->
+  ?loaded:unit ->
+  ?store_google:bool ->
+  ?save_referrer:bool ->
+  ?test:bool ->
+  ?verbose:bool ->
+  ?img:bool ->
+  ?track_links_timeout:int ->
+  ?cookie_expiration:int ->
+  ?upgrade:bool ->
+  ?disable_persistence:bool ->
+  ?disable_cookie:bool ->
+  ?secure_cookie:bool ->
+  ?ip:bool ->
+  ?property_blacklist:string list ->
+  ?name:string ->
+  unit ->
+  bool Lwt.t
```

Comment:
```suggestion
  unit Lwt.t
```

Since this function always returns `true`.

Reply:
But the code on "bs" works with a "Lwt.pick" in "bs_mixpanel.eliom" (https://github.com/besport/bs/pull/12543/files), did I have to change it?

Reply:
Yes, you need to the adjust the code accordingly.

Reply:
I tried to change the type of the returned value but it failed, the compilation deduce that the typed returned is "bool" and not unit:
<<
 Values do not match:
         val init :
           token:string ->
           ?api_host:string ->
           ?app_host:string ->
           ?autotrack:bool ->
           ?cdn:string ->
           ?cross_subdomain_cookie:bool ->
           ?persistence:string ->
           ?persistence_name:string ->
           ?cookie_name:string ->
           ?loaded:bool ->
           ?store_google:bool ->
           ?save_referrer:bool ->
           ?test:bool ->
           ?verbose:bool ->
           ?img:bool ->
           ?track_links_timeout:int ->
           ?cookie_expiration:int ->
           ?upgrade:bool ->
           ?disable_persistence:bool ->
           ?disable_cookie:bool ->
           ?secure_cookie:bool ->
           ?ip:bool ->
           ?property_blacklist:string list ->
           ?name:string -> unit -> bool Lwt.t
       is not included in
         val init :
           token:string ->
           ?api_host:string ->
           ?app_host:string ->
           ?autotrack:bool ->
           ?cdn:string ->
           ?cross_subdomain_cookie:bool ->
           ?persistence:string ->
           ?persistence_name:string ->
           ?cookie_name:string ->
           ?store_google:bool ->
           ?save_referrer:bool ->
           ?test:bool ->
           ?verbose:bool ->
           ?img:bool ->
           ?track_links_timeout:int ->
           ?cookie_expiration:int ->
           ?upgrade:bool ->
           ?disable_persistence:bool ->
           ?disable_cookie:bool ->
           ?secure_cookie:bool ->
           ?ip:bool ->
           ?property_blacklist:string list ->
           ?name:string -> unit -> unit Lwt.t
>>
Maybe I haven't understand what the function have to return

Reply:
It should work by adjusting the value passed as argument `loaded` to `Base.config`.